[DS-3615] Allows changing of value-pairs-name for language tag.

edusperoni opened the pull request:
https://jira.duraspace.org/browse/DS-3615
If this gets approved I'll port it to master.

Commit:
Allows changing of value-pairs-name for language tag.

AlexanderS commented on dspace-api/src/main/java/org/dspace/app/util/DCInput.java:
```diff
@@ -114,7 +114,8 @@ public DCInput(Map<String, String> fieldMap, Map<String, List<String>> listMap)
         if (language)
         {
-            valueLanguageList = listMap.get(LanguageName);
+            languageName = fieldMap.get("language-value-pairs-name");
```
Can we use the default value here, if no language-value-pairs-name is given?

AlexanderS replied:
Ah scratch this. I just saw that `DCInputsReader` is enforcing the language attribute.

AlexanderS commented on dspace-api/src/main/java/org/dspace/app/util/DCInput.java:
```diff
@@ -33,7 +33,7 @@
     
     /** the language code use for the input */
-    private static final String LanguageName = "common_iso_languages";
+    private String languageName = "common_iso_languages";
```
This can be `null`. No need for a value, that is never used.

edusperoni replied:
Good call, added it to the PR.

edusperoni committed:
Set languageName to null

Setting languageName to null, according to https://github.com/DSpace/DSpace/pull/1773

AlexanderS reviewed (approved):
Tested it and it works.
